pycheribuild/projects/build_qemu.py

```diff
@@ -338,7 +338,7 @@ def qemu_cheri_binary(cls, caller: SimpleProject, xtarget: CrossCompileTarget =
 
 
 class BuildCheriOSQEMU(BuildQEMU):
-    repository = GitRepository("https://github.com/CTSRD-CHERI/qemu.git", default_branch="cherios", force_branch=True)
+    repository = GitRepository("https://github.com/CTSRD-CHERI/qemu.git", default_branch="qemu-cheri", force_branch=True)
```

Comment:
Should be able to use the normal CHERI QEMU build now and remove this target?

Reply:
Gonna have it float around for now because I might point it to the morello branch tomorrow so I can play around with cherios-morello. But yes, soon!